### ISSUE TITLE
Multiple SKU - Uniqueness Validation Required

### ISSUE THREAD

**Issue by Repair-Desk** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#309]
I'm also encountering an annoying bug with multi-SKUs that I'll publish to the GitHub repo soon if I don't forget. A lot of items are saying the SKU is existing already while a few are allowing me to add the specific SKU. e.g. iPhone 6 screen has had SKU 1234 for Sentrix, I go to add multiple SKUs and add Sentrix 1234 for that same item that has existed, says SKU exists already. However a few other items allow me to add it and some don't. Super weird. Haha

**Comment by serfma** [comment]
https://youtu.be/BDrWidobGcE Video of the issue :)

**Comment by Repair-Desk** [comment]
Thanks for the message, we'll get back to you as soon as
possible!

On May 29, 2018 at 5:29 PM Repair-DeskPOS/RepairDesk-Bugs
wrote:

https://youtu.be/BDrWidobGcE ( https://youtu.be/BDrWidobGcE )
Video of the issue :)

--

You are receiving this because you authored the thread.

Reply to this email directly or view it on GitHub:

https://github.com/Repair-DeskPOS/RepairDesk-Bugs/issues/309#issuecomment-392991756
( https://github.com/Repair-DeskPOS/RepairDesk-Bugs/issues/309#issuecomment-392991756 )

Powered by UserVoice. ( http://www.uservoice.com/powered-by?utm_campaign=poweredby&utm_medium=email&utm_source=ticket&uv_company=RepairDesk&uv_contact_url=repairdesk.uservoice.com&uv_experience=email )

**Comment by serfma** [comment]
To add on that will help: I had created an item with the SKU for the new vendor, Injured Gadgets. "IP6p-Assem-Midgrade-White" and "IP6p-Assem-Midgrade-Black". I deleted it and started using multiple SKUs, however when I attempt to add it it says the SKU already exists which appears the item didn't actually delete from my inventory even though I can't find it.

**Comment by Repair-Desk** [comment]
Thanks for the message, we'll get back to you as soon as
possible!

On May 29, 2018 at 5:50 PM Repair-DeskPOS/RepairDesk-Bugs
wrote:

To add on that will help: I had created an item with the SKU
for the new vendor, Injured Gadgets. "IP6p-Assem-Midgrade-White"
and "IP6p-Assem-Midgrade-Black". I deleted it and started using
multiple SKUs, however when I attempt to add it it says the SKU
already exists which appears the item didn't actually delete from
my inventory even though I can't find it.

--

You are receiving this because you authored the thread.

Reply to this email directly or view it on GitHub:

https://github.com/Repair-DeskPOS/RepairDesk-Bugs/issues/309#issuecomment-392995129
( https://github.com/Repair-DeskPOS/RepairDesk-Bugs/issues/309#issuecomment-392995129 )

Powered by UserVoice. ( http://www.uservoice.com/powered-by?utm_campaign=poweredby&utm_medium=email&utm_source=ticket&uv_company=RepairDesk&uv_contact_url=repairdesk.uservoice.com&uv_experience=email )
